requirements.txt

```diff
@@ -26,11 +25,11 @@ PyYAML==3.13
 scikit-learn==0.20.2
 scipy==1.1.0
 SimpleITK==1.2.0
-six==1.10.0
+six>=1.10.0
 sklearn==0.0
 soupsieve==1.9.1
-tensorboard==1.12.2
-tensorflow==1.12.3
+tensorboard>=1.14.0
+tensorflow>=1.14.0
```

Comment:
I think this will by default install the latest version of each package (in this case TensorFlow). However, I think that newer versions of tensorflow (2.x.x) are not able to load the model in this repo because it is saved using tensorflow 1.x.x, which can lead to weird results sometimes. 

Is there a particular reason you're changing the requirements? I'd like to keep them fixed to the exact versions that were used in the experiments for reproducibility. 

If in your experience the pipeline does work with these requirements I would suggests we create a new requirements file for this. So the original `requirements.txt` remains unchanged and has the exact package versions from the experiment. Then we also create another requirements file with updated requirements that have been tested by people.

Reply:
pip did not resolve tensorflow 1.12.3 for me so I was unable to install it from pypi.org. I ended up installing the 1.14.0 version as that was the closest version that worked for me and it's also the oldest version on the modules env of the gpu cluster `TensorFlow/1.14.0-fosscuda-2019a-Python-3.7.2`. Perhaps you could consider affixing the version in the requirements.txt to that? In any case, I've added a commit that affixes the version.

Reply:
Sounds good! Might need to update the README.md with additional instructions though.